backend/src/tests/auth.test.js

```diff
@@ -0,0 +1,46 @@
+import { assert } from "chai";
+import { request } from "chai-http";
+import server from "../../server.js";
+
+describe("auth route tests", () => {
+  it("should login a user that exists", (done) => {
```

Comment:
Instead of testing against a real username and password in your DB or file, It is better to add a before hook to this test in which you will create a test user and then in the it block assert that this user can login.